check_links.py

```diff
@@ -0,0 +1,97 @@
+import os
+import urllib.request
+from bs4 import BeautifulSoup
+import markdown
+import re
+
+
+def get_html():
+    lst_html = []
+    # search path is hard coded with ref of this script locaiton
+    for root, dirs, files in os.walk(os.path.join(".","docs","build","html"), topdown=False):
+        for name in files:
+            if name[-4:] == "html":
+                filepath = os.path.join(root, name)
+                print(filepath)
+                lst_html.append((filepath, root))
+    return lst_html
+
+def get_md():
+    lst_md=[]
+    # search path is hard coded with ref of this script locaiton
+    for root, dirs, files in os.walk(".", topdown=False):
+        for name in files:
+            if name[-2:] == "md":
```

Comment:
We also have .rst files such as this: https://github.com/aimakerspace/PeekingDuck/blob/dev/docs/source/getting_started/04_benchmarks.rst